CMakeLists.txt

```diff
@@ -144,3 +146,28 @@ endif (BUILD_TESTING)
 ##
 # Adds custom uninstall command
 add_custom_target(uninstall_vc4cl "${CMAKE_COMMAND}" -P "cmake_uninstall.cmake")
+
+add_custom_command(
+  OUTPUT ${PROJECT_BINARY_DIR}/VC4CL.icd
+  COMMAND echo "${CMAKE_INSTALL_PREFIX}/lib/libVC4CL.so.1.2" > ${PROJECT_BINARY_DIR}/VC4CL.icd
+  )
+
+add_custom_target(generate_icd DEPENDS VC4CL.icd)
+add_dependencies(VC4CL generate_icd)
+install(FILES VC4CL.icd DESTINATION "/etc/OpenCL/vendors")
+
+if (BUILD_DEB_PACKAGE)
+	message(STATUS "build deb package...")
+
+	set(CPACK_GENERATOR "DEB")
+	set(CPACK_PACKAGING_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}")
+	set(CPACK_PACKAGE_NAME "vc4cl")
+	set(CPACK_PACKAGE_VERSION ${PROJECT_VERSION})
+	set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "armhf")
+	set(CPACK_DEBIAN_PACKAGE_DEPENDS "opencl-c-headers, ocl-icd(>=1.3)")
```

Comment:
Also depends on `vc4c`, since we build VC4CL with VC4C compiler enabled

Reply:
I also add `clang-3.9` to the dependency.
And, it seems that debian package name is required. So I modified `ocl-cid` to `ocl-icd-opencl-dev`.